packages/content-management/content_editor/src/components/editor_flyout_content.tsx

```diff
@@ -70,6 +70,7 @@ export const ContentEditorFlyoutContent: FC<Props> = ({
   const { euiTheme } = useEuiTheme();
   const [isSubmitting, setIsSubmitting] = useState(false);
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const isMounted = useRef(false);
```

Comment:
Note: the changes on this file are _not_ to fix the issue but I realized some React error thrown in the console because of a state update after the Table list view editor closes and decided to fix it 😊

Reply:
FWIW, this isn't necessary. The warning is removed in newer React versions https://github.com/reactwg/react-18/discussions/82. 
if you ask me, I'd prefer to keep the code a bit cleaner and I'd leave the warning.

Reply:
Oh ok I didn't know that. I will revert the changes in the file then.

> To sum up, we're removing this warning. 

They have been removing this console.log since Aug 19th 2021... hope it lands in main at some point!

Reply:
https://github.com/elastic/kibana/issues/138222